Arslan.cpp

```diff
@@ -0,0 +1,13 @@
+#include<iostream>
+using namespace std;
+
+int main(int argc, char const* argv[]) {
+    string username;
+  
+    cout << "Enter your username: ";
+    cin >> username;
+
+    std::cout << "Hello, " << username << std::endl;
```

Comment:
No need to write `std::cout` and `std::endl` if you already wrote `using namespace std;`